app/controllers/comment_controller.php

```diff
@@ -33,16 +46,16 @@ public function write()
         $page = Param::get('page_next','write');
 
         switch($page) {
-            case 'write':
+            case self::CURRENT_PAGE_WRITE:
                 break;
-            case 'write_end':
+            case self::REDER_PAGE_AFTER_WRITE:
```

Comment:
Please check [this](https://github.com/renzosunico/MyClassroom/pull/2/files#r39119381)